index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    rel="stylesheet">
+</head>
+<body>
+    <div class="main-container">
+        <nav class="nav-bars">
+            <nav class="logo">
+              <h2>DNG</h2>
+            </nav>
+            <nav class="menu">
+              <nav><img src="photo/navs/Union.png" alt=""></nav>
+            </nav>
+        </nav>
+   
+        <section class="About-me">
+            <div class="greeting">
+               <h1>I'm Duncan Glad to see you!</h1>
+            </div>
+
+            <div class="biography">
+              <p>I’m a software developer! I can help you design and build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+
+            <button class="link" type="submit">LET'S CONNECT</button>
+        
+            <div class="icon">
+               <img src="photo/logos/twitter.jpg" alt="">
+               <img src="photo/navs/Linkedin-icon.jpg" alt="">
+               <img src="photo/logos/medium.jpg" alt="">
+               <img src="photo/logos/github.jpg" alt="">
+               <img src="photo/logos/angellist.jpg" alt="">
```

Comment:
- [ ] Please the icons present an unordered list of elements please it would be semantically better to use `ul` to list the icons and they themself should be contained in the `li` tag which contains `a` tag